docs/ws-clients.md

```diff
@@ -0,0 +1,64 @@
+# Web Service Clients
+
+## Python Clients
+
+### Images import Python client
+
+An up-to-date version of said client can be founded on GitHub at the following URL : [https://github.com/OpenSILEX/phis-ws-clients/blob/master/python/images/post.py](https://github.com/OpenSILEX/phis-ws-clients/blob/master/python/images/post.py).
+
+Download this `post.py` script and change with the help of a text editor (e.g. Notepad++ if you are using Windows) its rows n° 17, 27, 28 and 58.
+
+#### Modification of the Python script
+Having installed [Python](https://www.python.org/) is required to run the `post.py` script.
+Here is the list of changes you need to make to this script in order to import your images to PHIS :
+
+**Row n° 17** :
+
+Replace the default host name `'http://localhost:8084/phis2ws/rest/'` by the host name associated to your occurence of PHIS.
+The URL of your documentation link should be of the form `'http://[...]/[occurence name]/api-docs/'`. The host name you need to provide on row 17 of `post.py` is this URL but with "api-docs" replaced by "rest" : `'http://[...]/[occurence name]/rest/'`.
+Do not forget the single brackets.
+
+**Row n° 27 & 28** :
+
+Rows 27 and 28 are used for indicating your own login, the one you use when connecting to PHIS via a web browser.
+
+On row 27, replace the second occurence of 'username' by your username between single brackets : it should be an email adress, e.g. `'guestphis@supagro.inra.fr'`.
+
+On row 28, either replace the second occurence of 'password' by your password between single brackets, either replace `hashlib.md5('password').hexdigest()` by an encrypted version of your password : a md5 checksum of your password (also between single brackets), e.g. `'1a79a4d60de6718e8e5b326e338ae533'` (this is the md5 encryption of "example").
```

Comment:
The user can add its password in clear and does not need to generate himself the md5 encryption of the password, the "hashlib.md5('password').hexdigest()" will generate the md5 of the password